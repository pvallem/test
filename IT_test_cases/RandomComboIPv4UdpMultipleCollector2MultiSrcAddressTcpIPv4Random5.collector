bidir_flow_info {
  key {
    src_address: "\001\002\003\001"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\263}O\275"
    dst_address: "\325\352\372w"
    src_port: 12955
    dst_port: 5122
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\000"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    flow_features {
      packet_count: 2
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\256V\215s"
    dst_address: "\324\214\317\205"
    src_port: 17295
    dst_port: 58677
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
