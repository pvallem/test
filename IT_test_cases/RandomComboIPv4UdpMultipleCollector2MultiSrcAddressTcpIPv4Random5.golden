flow_info {
  key {
    src_address: "\001\002\003\000"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200000040
}
flow_info {
  key {
    src_address: "\325\352\372w"
    dst_address: "\263}O\275"
    src_port: 5122
    dst_port: 12955
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "\001\002\003\001"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\256V\215s"
    dst_address: "\324\214\317\205"
    src_port: 17295
    dst_port: 58677
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV4
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
